docs/custom-telemetry.md

```diff
@@ -0,0 +1,43 @@
+## Custom Telemetry With Scarf’s HTTP API<a id="custom-telemetry-with-scarfs-http-api"></a>
+
+Scarf provides you the ability to collect custom telemetry from within your application by utilizing our API. To enable this you'll need a Scarf account and a File Package created. Enable Event Collection Only. 
+
+Once this has been done, you are able to send telemetry data and associate it with the File Package you just created. Scarf has published and maintains an example in [bash or shell](https://github.com/scarf-sh/scarf-shell/), included below is an example Python function developed by one of our customers.
```

Comment:
```suggestion
Once this has been done, you can send telemetry data and associate it with the Scarf package you just created via HTTP requests to your configured endpoint. Here is a real world example from Unstructured.io in Python:
```

Reply:
I'd also say perhaps we include the Scarf shell example at the end, and link to the exact file where the code is: https://github.com/scarf-sh/scarf-shell/blob/main/scarf.sh. But the focus is the Python example so let's mention that first.